addons/l10n_ar_website_sale/views/templates.xml

```diff
@@ -44,6 +44,9 @@
     </template>
 
     <template id="address_b2b" inherit_id="website_sale.address_b2b">
+        <xpath expr="//t[@id='t_address_b2b']" position="attributes">
+            <attribute name="t-if">res_company.country_id.code == 'AR' or is_view_active('website_sale.show_address_b2b')</attribute>
```

Comment:
@nboulif Is there a way we could generalize the condition and maybe put in standard already.  (a method we could overrride e.g.) because if mutliple localizations want this, they could override each other.   (although AR might be the only country where it is always required)